treestats_paper.tex

```diff
@@ -570,6 +583,8 @@ \subsection*{Site statistics}
     for the region of the genome between $a$ and $b$.
 \end{example}
 
+\krt{There is another approach, introduced by Molly Prz.  I'll see if I can dig up a description.  The approach
+described here is preferable, though.}
```

Comment:
I've recently found out that the original descriptions of the ABBA-BABA statistic had it being polarized if one took the equations literally, although reading the words it sounded like it was not. Do you know if it's supposed to be?

Reply:
This comment is in reference to how > 2 states at a Site are treated.  I am not sure this is important for now, and the comment is more for me to look up that older paper.